yarn.lock

```diff
@@ -671,9 +671,9 @@
   "resolved" "https://registry.npmjs.org/@next/react-refresh-utils/-/react-refresh-utils-11.1.2.tgz"
   "version" "11.1.2"
 
-"@next/swc-darwin-x64@11.1.2":
```

Comment:
I'm curious as to why this changed. 
If you know why please let me know, otherwise I ***think*** this is safe to ignore 😄

Reply:
Ok so I'm not sure about this one but it probably stems from committing from my windows machine and how yarn handles dependencies across operating systems. In general I think we don't need to push the yarn.lock file since open source contributors who use yarn can now use an import command that makes use of the package lock file. (assuming everyone on our team uses npm) 

source: https://stackoverflow.com/questions/44552348/should-i-commit-yarn-lock-and-package-lock-json-files/44904494